### ISSUE TITLE
FmtNotYetImplementedError when rendering a redirect

### ISSUE THREAD

**Issue by christiandmiddel** [sweble/sweble-wikitext#65]
I have ran into another issue when a page is rendered that contains a redirect.
```
@Override
    public void visit(WtRedirect n) {
        // TODO: Implement
        throw new FmtNotYetImplementedError();
    }
```

To make a long story short I will put another pull request for further discussion...

**Comment by christiandmiddel** [comment]
I am unsure if this is fine for you. It would perfectly fit my use case.
What I am completely unsure is there a better way to create the PageTitle object to be injected into the callback?

**Comment by hannesd** [comment]
The you did it looks correct. Merged in 99ebf4054f3698da77b685364f6e7630e137e1e5.
